org.eclipse.jdt.junit.core/plugin.xml

```diff
@@ -106,6 +106,18 @@
          <runtimeClasspathEntry pluginId="org.eclipse.jdt.junit.runtime" />
       </kind>
    </extension>
+   <extension
+         point="org.eclipse.jdt.junit.internal_testKinds">
+      <kind
+            id="org.eclipse.jdt.junit.loader.junit6"
+            displayName="%testKind.junit6"
+            finderClass="org.eclipse.jdt.internal.junit.launcher.JUnit5TestFinder"
+            loaderPluginId="org.eclipse.jdt.junit5.runtime"
+            loaderClass="org.eclipse.jdt.internal.junit5.runner.JUnit5TestLoader">
+         <runtimeClasspathEntry pluginId="org.eclipse.jdt.junit5.runtime" />
```

Comment:
If we merge https://github.com/eclipse-jdt/eclipse.jdt.ui/pull/2535, we can refer to the new bundle here.